packages/vuetify/src/mixins/data-iterable.js

```diff
@@ -231,6 +231,29 @@ export default {
       if (this.pageStart >= this.itemsLength) {
         this.resetPagination()
       }
+      const itemsRemoved = {}
```

Comment:
Alternative solution

```
const newItems = new Set(this.items.map(item => getObjectValueByPath(item, this.itemKey)))
const selection = this.value.slice().filter(item => newItems.has(getObjectValueByPath(item, this.itemKey)))

if (selection.length !== this.value.length) {
  this.$emit('input', selection)
}
```

Reply:
I didn't knew if using `Set` would be okay. It's much cleaner, I agree. Will push an update soon. Thanks for the review!